binderhub/repoproviders.py

```diff
@@ -111,6 +126,38 @@ def has_higher_quota(self):
                 return True
         return False
 
+    def repo_config(self, settings):
```

Comment:
should we make this a property or keep an explicit method?

Reply:
I think it can't be a property if it's taking arguments.

Reply:
I am happy with it being a method.